### DISCUSSION TITLE
Using custom color scheme in vaex.viz.histogram

### DISCUSSION THREAD

**Discussion by DanielvdC** [vaexio/vaex#1728] [Q&A] [answered]
Hi,

I am trying to visualise a large amount of timestamps of deliveries made in the past two years (+- 3 billion data points) where each data point has a value associated with it regarding its timing: 'Too early', 'On time' or 'Too late'.

I successfully plotted the data points with the viz.histogram function, but is it possible to color datapoints regarding to their value? I would like them to be yellow, green and red, respectively. Otherwise, is there perhaps another way of visualising that would better suit my needs?

Thanks in advance!

**Comment by JovanVeljanoski** [comment, marked as answer]
Well,  you can't really plot 3 billion points anyway (if I understand you correctly)..  That's why we do histograms. 
Maybe I am misunderstanding what you are trying to do. 

You can easily overplot 3 histograms associated with each of your 3 flags. Would that be helpful? 
You can do it with something like
```python
df.viz.histogram(df.timestamps, selection=df.flag == 'Too early') # And similar for the other 2 choices. 
```

**Reply by DanielvdC** [reply]
This will do the trick indeed. Thanks!

**Comment by maartenbreddels** [comment]
Maybe stacked histograms?
